subscription.tf

```diff
@@ -1,4 +1,6 @@
 module "observe_cloudwatch_logs_subscription" {
+  name = "${var.name}-logs"
```

Comment:
How does this address the issue? I'd assume the way to apply the module multiple times against the same environment would be to provide a unique `name`. How does adding this suffix resolve the conflict?

Reply:
we didn't pass in a name before, so observe_cloudwatch_logs_subscription always uses the same default name

Reply:
I see, I misread this as appending `-logs` and not adding a `name`:

https://github.com/observeinc/terraform-aws-cloudwatch-logs-subscription/blob/897508aa88b9c594cc8dbd8275249507dbd8dd5b/variables.tf#L5

Any change that results in replacement of resources should be considered breaking. If you wanted to add this in a non-breaking way, you could add a new variable, e.g., `var.cloudwatch_logs_name`, where `default = null`.

Reply:
I see, then we can wait until a major version bump to change this. I'll add the variable and a comment then